specification/ai/Face/models.session.tsp

```diff
@@ -166,7 +166,7 @@ union ImageType {
 }
 
 @doc("The liveness classification for target face.")
-model LivenessOutputsTarget {
+model LivenessDecisionTarget {
```

Comment:
This is a retro breaking change. Shouldn't you use @rename versioning decorator?

Reply:
I thought it's okay since it's mostly only affecting documentation. We are not changing the actual name inside payload. 
BTW, this is some feedback from C# SDK review, but I think it worth to change it at REST level for better naming.

Reply:
`@rename` should work for intention to change it going forward.

Reply:
Let's wait for the final design, we may need to rebuild a lot of models after the change.